source/_docs/scripts.markdown

```diff
@@ -753,13 +755,13 @@ automation:
 
 ## Parallelizing actions
 
-By default, all sequences of actions in Home Assistant run sequentially. This
-means the next action is started after the current action has been completed.
+By default, all sequences of {% term actions %} in Home Assistant run sequentially. This
+means the next {% term action %} is started after the current action has been completed.
 
 This is not always needed, for example, if the sequence of actions doesn't rely
 on each other and order doesn't matter. For those cases, the `parallel` action
-can be used to run the actions in the sequence in parallel, meaning all
-the actions are started at the same time.
+can be used to run the {% term actions %} in the sequence in parallel, meaning all
```

Comment:
It works also on actions or only action?

Reply:
Both. The alias is set.
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/d6eb6c78-7b55-4df4-8099-d7fee96ac1e4)